OpenEdXMobile/build.gradle

```diff
@@ -181,7 +184,7 @@ dependencies {
     provided 'org.roboguice:roboblender:3.0.1'
 
     // For the optional Nullable annotation
-    compile 'com.android.support:support-annotations:23.3.0'
+    compile 'com.android.support:support-annotations:25.0.1'
```

Comment:
I tried keeping this lib and downgraded the other support libs i.e.

> compile 'com.android.support:support-v4:23.3.0'
compile 'com.android.support:recyclerview-v7:23.3.0'
compile 'com.android.support:cardview-v7:23.3.0'
compile 'com.android.support:appcompat-v7:23.3.0'
compile 'com.android.support:design:23.3.0'

and everything worked fine. But since I had to downgrade those and it didn't make any sense to upgrade only this lib (and more importantly none of these libs are required to update for firebase), I've now kept their version same as on `master`.